.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,53 @@
+on: workflow_dispatch
```

Comment:
nit: define a name. Same for the other workflows.

Reply:
Will do in next PR